frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -89,7 +83,7 @@
         img-bounds    (bounds-for-zoom point size bounds img-size)
         ;; Note, top layer, last in the list, must be first in our search string:
         request-id    (gensym)
-          had-insecure? (->> db :map :active-layers (some #(is-insecure? (:server_url %))))
+        had-insecure? (->> db :map :active-layers (some #(is-insecure? (:server_url %))))
```

Comment:
Ignore for now, but just for context -- we might be able to finally remove this at some point.  The gist is, browsers don't like making http ajax requests from a https page (or a bit more confusingly for me, even https from an http page), so we had this workaround because there was always some rogue GA or CSIRO layer still on http.  That _may_ finally be at an end.  (Like I said, leave it in, I'm just talking out loud)